packages/core/admin/admin/src/components/LeftMenu.tsx

```diff
@@ -1,82 +1,38 @@
 import * as React from 'react';
 
-import {
-  Box,
-  Divider,
-  Flex,
-  FocusTrap,
-  Typography,
-  MainNav,
-  NavBrand,
-  NavCondense,
-  NavFooter,
-  NavLink,
-  NavSection,
-  NavSections,
-  NavUser,
```

Comment:
Does this mean we can remove all these from the design system?

Reply:
Yeah they are not used anymore in the CMS, I am gonna create a jira in our board to remove them from the DS